[BUG] Build fails on armv7 targets

**Describe the bug**
Trying to compile this on a Raspberry Pi Model 3B running Raspbian causes a compile error in the expansion of the WindowExt derive macro. The same error occurs when cross-compiling from my AMD64 desktop running Debian Buster using the *armv7-unknown-linux-gnueabihf* target. On both platforms, the stable branch of Rust was used.

**Compiler output**
This error appears in every location where WindowExt is derived:
```
error[E0308]: mismatched types
  --> src/window.rs:34:31
   |
34 | #[derive(WidgetExt, GroupExt, WindowExt, Debug)]
   |                               ^^^^^^^^^
   |                               |
   |                               expected `u64`, found `u32`
   |                               expected `u64` because of return type
   |                               help: you can convert an `u32` to `u64`: `WindowExt.into()`
   |
   = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

--- a/fltk-derive/src/window.rs
+++ b/fltk-derive/src/window.rs
@@ -153,7 +153,7 @@ pub fn impl_window_trait(ast: &DeriveInput) -> TokenStream {
                         target_os = "netbsd",
                         target_os = "openbsd",
                     ))]
-                    return winid.x_id;
+                    return winid.x_id as RawHandle;
                 }
             }
 

